### PR TITLE
6.7.x

### LINKED ISSUE #6: Fix wrong Iso Code "fr" in TSConfig
There is the wrong Iso-2-Code in the TSConfig of the extension.

### PR DESCRIPTION
**Pull request by Kephson**
fixes #6, removed deprecated TypoScript for TYPO3 9.5

### CODE REVIEW

File: ext_localconf.php [hunk 1]
```diff
@@ -2,12 +2,19 @@
 defined('TYPO3_MODE') or die();
 
 $initialize = function ($extKey) {
+    $isVersion9Up = \TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 9000000;
```
Comment (manuelselbach) [inline ext_localconf.php]: Is there a reason not to use [version_compare()](https://www.php.net/manual/de/function.version-compare.php) here?

File: ext_localconf.php [hunk 2]
```diff
@@ -2,12 +2,19 @@
...
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addTypoScriptSetup(
         '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $extKey . '/Configuration/TypoScript/Extbase/setup.txt">'
     );
-    \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
-        '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $extKey . '/Configuration/PageTSconfig/Suggest.tsconfig">'
-    );
+    if ($isVersion9Up) {
+        \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
+            '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $extKey . '/Configuration/PageTSconfig/Suggest_9.tsconfig">'
```
Comment (manuelselbach) [inline ext_localconf.php]: I would suggest to go the other way round and use `Suggest_prior_9.tsconfig` and leave `Suggest.tsconfig` as the file `Suggest.tsconfig` will last longer in this extension. What do you think?

**Review by manuelselbach — CHANGES_REQUESTED** [review]

File: ext_localconf.php (hunk 2 above)
Comment (Kephson) [reply ext_localconf.php]: Hi Manuel, that sounds good, doing it the other way, to support newer versions by default and the older ones with a special part.

**Comment by Kephson** [comment]
Hi, you could find the updates in 526e5487e9854c9954f25b7f8718645d207c873d

**Comment by manuelselbach** [comment]
Hi again ☺️ 

Thanks for the fast update.

Did you see my question in ext_localconf.php (line 5)?
May I ask you to provide an explanation?

File: ext_localconf.php (hunk 1 above)
Comment (Kephson) [reply ext_localconf.php]: Hi, sorry didn't recognized this... I'm normally using it in this way...

Comment (manuelselbach) [reply ext_localconf.php]: I would suggest to use `version_compare` here and use it directly in the if e.g.:
```
if (version_compare(TYPO3_branch, '9.5', '>=')) {
```
And we should mention the version where the feature got introduced to not break anything ☺️ 

If I have a look at the ChangLog for v9, the expression language was introduced with 9.4, IIUC: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.4/Feature-85828-MoveSymfonyExpressionLanguageHandlingIntoEXTcore.html

Comment (Kephson) [reply ext_localconf.php]: Hi, thanks for the fast reply, now I could understand it.
I changed it now to version_compare with TYPO3 branch.
See e35da1d

Comment (manuelselbach) [reply ext_localconf.php]: Looks good so far. I'm asking myself whether we need this line still:

`$isVersion9Up = \TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 9000000;`

**Comment by Kephson** [comment]
all changes done: 7ade91469da338f5b7e771904d1dcd79cb3325d6

**Review by manuelselbach — APPROVED** [review]

**Pull request merged**